src/main/java/io/kestra/plugin/azure/messaging/eventhubs/service/consumer/CheckpointStoreType.java

```diff
@@ -0,0 +1,8 @@
+package io.kestra.plugin.azure.messaging.eventhubs.service.consumer;
+
+/**
+ * Supported checkpoint type.
+ */
+public enum CheckpointStoreType {
+    BLOB;
```

Comment:
An enumeration with only one item seems useless, I'll remove it and add it later when we support multiple checkpoint store if needed